Scripts/script-Cut.yml

```diff
@@ -0,0 +1,36 @@
+commonfields:
+  id: cae96081-c034-46c1-81f6-b4d0a52dc51c
+  version: 38
+name: Cut
+script: |-
+  args = demisto.args()
+
+  value = args.get('value')
+  fields = args.get('fields')
+  delim = args.get('delimiter')
```

Comment:
I would verify input correctness:
- types (`value` and `delimiter` should be string, `fields` should be csv numbers)
- indexes in range